src/handleAction.js

```diff
@@ -1,19 +1,18 @@
 import isFunction from 'lodash/isFunction';
 import identity from 'lodash/identity';
 import isNil from 'lodash/isNil';
-import isSymbol from 'lodash/isSymbol';
+import includes from 'lodash/includes';
+import { ACTION_TYPE_DELIMITER } from './combineActions';
 
-export default function handleAction(type, reducers, defaultState) {
-  const typeValue = isSymbol(type)
-    ? type
-    : type.toString();
+export default function handleAction(actionType, reducers, defaultState) {
+  const actionTypes = actionType.toString().split(ACTION_TYPE_DELIMITER);
```

Comment:
This will also fail and needs `babel-polyfill`. Instead this would also work with `lodash/includes`:

```js
if (!includes(actionType, action.type) {
```